cli/command_snapshot_create.go

```diff
@@ -19,6 +19,39 @@ import (
 )
 
 const (
+	snapshotCreateHelp = `Creates a snapshot from the given path.
+
+A snapshots is a point in time backup of the given <path>. Snapshots are organized by source.
+Each user@host:/path is considered a unique source, and can have policies applied to them.
+
+Example: $ kopia snapshot create /home/user
```

Comment:
can you try adding markdown around code smaples (triple backtick):

```$ kopia snapshot create /home/user```

This would still look decent in `--help` but will result in nicely formatted website help.

Reply:
Sure. I've found I can do something like 
```
`Example:
` + "```" + `
$ kopia snapshot list
` + "```" + `
Continuing....`
```
And I get this output on the cli 
```
``` $ kopia snapshot list ```
```
Would using a single backtick work for one liners? I think that would be a good compromise